README.md

```diff
@@ -89,6 +89,14 @@ Install using [Scoop](https://scoop.sh)
 $ scoop install dbmate
 ```
 
+**GO Install**
+
+Remember to add `~/go/bin` into your `$PATH` if you decide to install dbmate this way.
```

Comment:
`~/go/bin` is not correct in all cases.  A more generally correct recommendation would be `$(go env GOPATH)/bin`.